katran/lib/bpf/balancer_kern.c

```diff
@@ -220,18 +213,7 @@ static inline int process_l3_headers(struct packet_description *pckt,
       // contains ip options, and we dont support em
       return XDP_DROP;
     }
-
```

Comment:
If have F_SRC_ROUTING but not INLINE_DECAP, then we do not decrement ttl and potentially introduce infinite forwarding of packets between two hops

Reply:
also currently it is not possible to have LPM_SRC_LOOKUP (w/o this F_SRC_ROUTING is nop) w/o INLINE_DECAP: https://github.com/facebookincubator/katran/blob/master/katran/lib/bpf/balancer_consts.h#L246